plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/worker/LintErrorResult.kt

```diff
@@ -0,0 +1,13 @@
+package org.jlleitschuh.gradle.ktlint.worker
+
+import java.io.File
+import java.io.Serializable
+
+internal data class LintErrorResult(
+    val lintedFile: File,
+    val lintErrors: List<Pair<SerializableLintError, Boolean>>
```

Comment:
This depth of generics is kinda a code smell when inside a data class, but I can guess why you're doing it.

Can you at least document what the `Boolean` indicates here in context?

Reply:
Fixed via 3ce901a